sdk/advisor/Azure.ResourceManager.Advisor/src/autorest.md

```diff
@@ -5,7 +5,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 ``` yaml
 
 azure-arm: true
-generate-model-factory: false
+generate-model-factory: true
```

Comment:
We could just remove this config, the default value of this config is actually true and this is the reason why we must have it to be false.